### PR TITLE
Returning String instead of list based on test requirement in nvme.py

### PR DESCRIPTION
**Pull request by Naresh-ibm**
current method is returning list but it is easy to handle if the controller name is in string format. Hence modified accordingly

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
before fix
-----------
```

>>> nvme.get_controller_name('0563:20:00.0')
['nvme0']
```

after fix
-----------

```
>>> nvme.get_controller_name('0563:20:00.0')
'nvme0'
>>>
```

File: avocado/utils/nvme.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def get_controller_name(pci_addr):
     """
     if pci_addr in pci.get_pci_addresses():
         path = f"/sys/bus/pci/devices/{pci_addr}/nvme/"
-        return os.listdir(path)
+        return "".join(os.listdir(path))
```
Comment (PraveenPenguin) [inline avocado/utils/nvme.py:50]: some how i like older way it self as have multiple device than accessing list will be easy

Comment (Naresh-ibm) [reply avocado/utils/nvme.py:50]: @PraveenPenguin we will have only one controller with corresponding pci_address.
each nvme drive will have specific pci_address. Hence the output is expected to be one device only.

Comment (Naresh-ibm) [reply avocado/utils/nvme.py:50]: logs for your reference

```
[root@sys]# lspci -nnD
0563:20:00.0 Non-Volatile memory controller [0108]: Samsung Electronics Co Ltd NVMe SSD Controller PM173Xa [144d:a825]
0567:38:00.0 Non-Volatile memory controller [0108]: Samsung Electronics Co Ltd NVMe SSD Controller PM173Xa [144d:a825]
[root@sys]#
[root@sys]# pwd
/sys/bus/pci/devices/0563:20:00.0/nvme
[root@sys]# ls
nvme0
[root@sys]# 


[root@sys]# cd ../../0567\:38\:00.0/nvme/
[root@sys]#
[root@sys]# pwd
/sys/bus/pci/devices/0567:38:00.0/nvme
[root@sys]# ls
nvme1
[root@sys]#


```

Comment (PraveenPenguin) [reply avocado/utils/nvme.py:50]: thanks, @Naresh-ibm for info, can you please address static checks

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**